demosplan/DemosPlanCoreBundle/Addon/AddonRegistry.php

```diff
@@ -33,7 +32,7 @@ public function __construct()
     }
 
     /**
-     * @param Definition[] $addonInfos
+     * @param AddonInfo[] $addonInfos
```

Comment:
This is not correct. In `LoadAddonInfoCompilerPass` an array of `Definition` is set

Reply:
done in [173a301](https://github.com/demos-europe/demosplan-core/pull/2928/commits/173a301a5acfd2b30cd1517eb9555b8e4f1f5721)